### ISSUE TITLE
Require semicolons

### ISSUE THREAD

**Issue by WebRuin** [thomaspark/flexboxfroggy#18]
Great Job!!! this is awesome.

I think your should require semicolons in order to to solve the levels or at least a warning.

Thanks

**Comment by thomaspark** [comment]
Agree, though technically valid to leave off the semicolon for the last rule in a ruleset, it would be good to give the suggestion. Ties into discussion in #15 about a warning/hint system.

**Comment by L0rdV4der** [comment]
I'm also interested in the semicolon deal new programmers and veterans alike tend to accidentally leave out semicolons. If not a requirement then a helpful reminder that the person forgot would be helpful in the development of programmers learning from flexbox froggy.
